Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -167,6 +167,15 @@ - (void)setSelection:(RCTTextSelection *)selection
   }
 }
 
+- (void)setTextContentType:(NSString*)type
+{
+  #if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
+    if (@available(iOS 10.0, *)) {
+        self.backedTextInputView.textContentType = [type isEqualToString:@"none"] ? @"" : type;
```

Comment:
According to apple's doc the default value is nil, so I guess we should use that instead of an empty string.

Reply:
Nevermind, just read the part about it in your test plan, maybe just add a comment to explain why we use the empty string.

Reply:
It seems like for password inputs, it still defaults to `textContentType` equal to `password`, even if you haven't explicitly set that value. So setting an empty string will let you disable autofill for passwords. I can add a comment in the obj-c code :+1: